config/production.json

```diff
@@ -1,4 +1,7 @@
 {
+  "prettify": true,
```

Comment:
production does not need to be pretty, right? (this might also be cancelled out by optimize...?)